index.md

```diff
@@ -2,25 +2,44 @@
 site: sandpaper::sandpaper_site
 ---
 
-This is a GitHub skill-up for Carpentries Maintainers and lesson developers.
-It is designed primarily as a resource for skill-up leaders,
-but may also be useful for self-directed learning.
+This skill-up teaches some of GitHub's features and the skills to use them for more effective (lesson) repository maintenance.
+It is designed primarily as a resource for skill-up leaders, but may also be useful for self-directed learning.
+The skill-up is taught with example lesson repositories in GitHub but the principles and skills covered are transferable to other repository hosting systems such as GitLab.
 
-We recommend that skill-up leaders use [the Instructor View](./instructor/index.html) when using the site.
+We recommend that skill-up leaders use [the Instructor View](./instructor/index.html) when visiting the site.
 
 ## Target Audience
+This skill-up is aimed at Carpentries community members who want to use GitHub’s features to manage their lesson project more effectively. 
 
-The skill-up is aimed at people with a basic familiarity with the GitHub interface,
-and the process of editing files and 
-committing changes to a repository via the GitHub web interface.
-It will be particularly relevant to people maintaining/developing
-a lesson using The Carpentries lesson infrastructure
-([the Workbench](https://carpentries.github.io/workbench/).
+It is suitable for people who:
 
-## Objectives
+* Are familiar with the folders and files of a Carpentries lesson repository.
```

Comment:
I don't know that this is a hard core requirement. We tend to explain the structure of the lesson repo template as we work through the skill-up demos.